docs/release/3.4/release-3-4-17.md

```diff
@@ -0,0 +1,321 @@
+OSG Software Release 3.4.17
+===========================
+
+**Release Date**: 2018-08-08
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   OSG 3.4 Repository
+    -   [Singularity 2.6.0](https://github.com/singularityware/singularity/releases/tag/2.6.0): Bug fix release
+        -   This release includes an extra option called "underlay" that is not
+            in the upstream release nor enabled by default but is supported by
+            OSG and recommended to be enabled instead of the "overlay" option.
+            [See the documentation for details.](https://opensciencegrid.org/docs/worker-node/install-singularity/#configuring-singularity)
```

Comment:
Personally I think this is still a little wordy and repeats most of what's said in the documentation so I think it can be trimmed down. But if you and @DrDaveD decide to keep it, that's fine with me, too.